### PR TITLE
LPD-30076 Fix LDAP#LDAPSecurityCredentialsAreHidden

### PR DESCRIPTION
**Pull request by manuelecastro**
Related issue: [LPD-30076](https://liferay.atlassian.net/browse/LPD-30076)

Since `OcularAssertElementImage` is broken, I'm refactoring the test by instead of asserting by image, now use the path to ensure the input type is password and by checking if the _Security Credential_ input is replaced by the placeholder(`TEMP_OBFUSCATION_VALUE`) after adding a new LDAP server and editing it.

### CODE REVIEW

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:relevant

**Comment by manuelecastro** [comment]
ci:test:relevant

**Comment by manuelecastro** [comment]
ci:test:security

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase [hunk 1]
```diff
@@ -598,7 +598,6 @@ definition {
 	@description = "This is a use case for LPS-73381. LDAP security credentials are obscured when entered."
 	@priority = 3
 	test LDAPSecurityCredentialsAreHidden {
-		property portal.upstream = "quarantine";
```
Comment (ChrisKian) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase]: Looks like you remove from quarantine here, only to re-add it in the next commit.  And I don't see the test run in the security suite.  Could you take a look at this property and make sure you have everything correct?

**Comment by manuelecastro** [comment]
ci:test:security

**Comment by manuelecastro** [comment]
ci:test:security

**Comment by manuelecastro** [comment]
ci:report:53143399

**Comment by manuelecastro** [comment]
[LDAP#LDAPSecurityCredentialsAreHidden](https://testray.liferay.com/#/project/34754/routines/34756/build/53143399/case-result/53146274) is now **PASSING**

The remaining failures are not related to the changes. :)

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase (hunk 1 above)
Comment (manuelecastro) [reply portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/ldap/LDAP.testcase:601]: I messed up while committing this, my bad :sweat_smile: . I pushed again and ran the security suite to check the test

**Comment by ChrisKian** [comment]
ci:test:sf

**Comment by ChrisKian** [comment]
Looks good @manuelecastro!  Once the SF test passes, then we'll just need @stian-sigvartsen to approve the changes before forwarding.

**Comment by manuelecastro** [comment]
Great! Thank you for the review @ChrisKian ! :tada:

**Comment by manuelecastro** [comment]
Hi @ChrisKian and @stian-sigvartsen ,

I was checking the files and saw that there were some unused code left after my changes, so I decided to remove them as well. I'll run the tests again just to be sure that everything is working fine. 

Thank you :)

**Comment by manuelecastro** [comment]
ci:test:sf

**Comment by manuelecastro** [comment]
ci:test:security

**Comment by manuelecastro** [comment]
ci:report:56608517

**Comment by manuelecastro** [comment]
[LDAP#LDAPSecurityCredentialsAreHidden](https://testray.liferay.com/#/project/34754/routines/34756/build/56608517/case-result/56624696) **PASSED** :heavy_check_mark: 

Remaining failures are not related to the changes. :+1:

**Comment by stian-sigvartsen** [comment]
ci:forward:force

**Comment by stian-sigvartsen** [comment]
Good job @manuelecastro !

**Review by stian-sigvartsen — APPROVED** [review]

**Pull request closed without merging**